content/outcomes/cloud-native-development-in-practice/_index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Cloud Native Development in Practice
+weight: -1
+layout: single
+description: |
+    Build a Cloud Native App.
+    Run it on TAS.
+    Scale it on TAS.
+    Then evolve it to Microservices.
+team:
+  - VMware Tanzu Labs
+---
+
+## Why is this learning path important?
+
+This learning path will take you on a journey of building a
+greenfield blocking web application from scratch,
```

Comment:
Could just be me, but what is a "blocking" we application? Consider linking to an explainer?

Reply:
TL;DR - _blocking_ architecture styles use _synchronous_ [request/response pattern]( [request/response](https://en.wikipedia.org/wiki/Request%E2%80%93response)), a request is made, and a "caller"'s execution is _blocked_ from doing more work until the response is received.  

Much of the web is built this way using HTTP, as well as legacy applications using [RPC](https://en.wikipedia.org/wiki/Remote_procedure_call) protocols (such as [RMI](https://en.wikipedia.org/wiki/Distributed_object_communication) and [IIOP](https://en.wikipedia.org/wiki/General_Inter-ORB_Protocol)).

This is contrasted with _non-blocking_ architectures, where a _caller_ is not blocked from doing work while downstream work may be executed asynchronously.

Given the growing popularity of non-blocking architectures, many students of the previous PAL had the impression coverage of both blocking and non-blocking styles would be covered, and why we explicitly called out the architecture style scope limitation for this learning path.

@anthonyvetter if you think after reviewing this we should put more in the introduction background, I can add a short section to better describe it.

Reply:
Removed "blocking" from index page, as architecture styles section may be enhanced with non-blocking solutions (event or reactive).

Enhanced scaling and hardening to be more explicit with request/response pattern.